### DISCUSSION TITLE
Is there the ability to generate  RSS feed from a collection?

### DISCUSSION THREAD

**Discussion by ajtatum** [fr0tt/benotes#52] [Ideas]
I currently use Raindrop, but this looks like an awesome replacement. The only thing is that there isn't any about RSS feeds. For example, in Raindrop I have a collection of links I'm gonna include in my newsletter. The collection in Raindrops can "produce" an RSS feed so the newsletter is mostly automatic as it just pulls in the RSS items and when I'm done, I move the links out of the collection. Is this something I can do with Benotes?

**Comment by fr0tt** [comment]
Hi there,

there is no out of the box solution/button to export a collection as a rss file. 
However you can export it as json with `php artisan export` command and then use a tool to convert json into rss.
If that isn't good enough, you could also create a rss feed programmatically by feeding it data/the collection via the [rest api](https://github.com/fr0tt/benotes/blob/master/api.md).
